src/extension-runners/firefox-android.js

```diff
@@ -384,6 +387,29 @@ export class FirefoxAndroidExtensionRunner {
     await adbUtils.amForceStopAPK(selectedAdbDevice, selectedFirefoxApk);
   }
 
+  async adbArtifactsDir(removeArtifacts?: boolean) {
+    const {
+      adbUtils,
+      selectedAdbDevice,
+    } = this;
+
+    const foundDirectories = await adbUtils.checkOrCleanArtifacts(
+      selectedAdbDevice, removeArtifacts
+    );
+
+    if (foundDirectories) {
+      if (removeArtifacts) {
+        log.info('Old artifacts directories starting with web-ext-artifacts ' +
+                `removed on ${selectedAdbDevice} device`);
```

Comment:
I think that this message could just say saomething like `Old web-ext artifacts have been found and removed from ${selectedAdbDevice}`.

Also, you need to make this a template string (See [doc page on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Expression_interpolation)) so that the `${...}` parts are going to be replaced with the related value.

Reply:
I am not sure but this is a template string and it is replacing the ${...} part